src/wizard/android/index.md

```diff
@@ -9,6 +9,36 @@ type: framework
 
 Sentry captures data by using an SDK within your application’s runtime. These are platform-specific and allow Sentry to have a deep understanding of how your app works.
 
+### Auto-Installation With the Sentry Android Gradle Plugin
+
+To install the plugin, please update your build.gradle file as follows:
+
+```groovy
```

Comment:
IIRC, tabs don't work for the Wizard, we need to sections for Groovy and Kotlin DSL